### DISCUSSION TITLE
My application has not been approved.

### DISCUSSION THREAD

**Discussion by SmallCoral** [community/community#38964] [GitHub Education]
![image](https://user-images.githubusercontent.com/70262298/201507736-259992b2-5bce-4ae9-a566-d1273736684d.png)

My application has not been approved. I am a Chinese student. Due to the special network environment in China, I have to use VPN to apply all the time.

**Comment by samus-aran** [comment]
Hi @SmallCoral 👋 

We support remote learners from across the global however we do require that students modify their browser's settings to enable location access. This has now become mandatory when applying. Although you may be using a VPN, by allowing us to utilise your browser location will provide a more accurate application submission using geolocation. 👍 

Please do try again and let us know how you get on.
